torch/_dynamo/variables/tensor.py

```diff
@@ -190,23 +190,23 @@ def dynamic_getattr(self, tx, name):
             # Which is incorrect, and violates the invariant that all sources should be eval()-able against the scope.
             _input_associated_real_value = eval(self.source.name(), scope)
         except Exception as exc:
-            raise NotImplementedError() from exc
+            return
```

Comment:
Returning `None` seems wrong since the call above is expecting to catch NotImplementedError.